### DISCUSSION TITLE
How to use Logic Gates? OR, AND, NOT.

### DISCUSSION THREAD

**Discussion by revaru** [BEEmod/BEE2-items#4196] [Tech Support / Q&A] [answered]
I put 2 buttons and want them both to be able to open the door, but not when both are pressed, but one by one. How do I connect logic_or and where do I place it to work?

**Comment by CubieJudy** [comment, marked as answer]
https://steamcommunity.com/sharedfiles/filedetails/?id=150558007

I use this for my logic gate knowledge when I was confused

**Comment by revaru** [comment]
Thanks! I understand after reading second time! Thanks.
